### PR TITLE
Adding link to the style guide

### PR DESCRIPTION
**Pull request by fidian**
This links out to the newly created style guide for Tests-Always-Included.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ A guide to contributing to opentoken.
     * Following steps may not be needed if forked properly, but in case you can't get to the upstream server.
         1. Configure the upstream server by running `git remote add upstream <repo address>` or editing your `.git/config` under `[remote "upstream"]`, changing `url`.
 3. Create a branch off master `git checkout -b <branch name>`. You don't want to do any work off master as this is where you will pull in upstream changes and merge them into your branches. Also is a good idea to push this remotely `git push -u origin <branch name>`.
-4. Write your code as normal.
+4. Write your code as normal.  Be sure to follow the [Tests-Always-Included Style Guide](https://tests-always-included.github.io/style-guide/)
```
Comment (GoogilyBoogily) [inline CONTRIBUTING.md]: This has two spaces after the first sentence. The other sentences only have one.

File: doc/style-guide.md [hunk 2]
```diff
@@ -1,4 +0,0 @@
-Naming
-------
-
-* Functions that return promises should use `...Async()` as a suffix.
```
Comment (lnagle93) [inline doc/style-guide.md]: Why was this line removed?

Comment (fidian) [inline doc/style-guide.md]: The file was removed.  Instead we point to the style guide site.

File: CONTRIBUTING.md (hunk 1 above)
Comment (fidian) [inline CONTRIBUTING.md]: Ask @kyle-long about the entomology of the two spaces and let me know which one is the correct way of doing things.

Comment (AbsentSemicolon) [inline CONTRIBUTING.md]: I think one space is generally accepted with online things.

**Comment by AbsentSemicolon** [comment]
Reviewed and looks good.
